server/internal/infrastructure/mongo/mongodoc/workspace.go

```diff
@@ -7,30 +7,44 @@ import (
 )
 
 type WorkspaceMemberDocument struct {
```

Comment:
I thought there would be compatibility issues with members becoming independent structs, but was the member already stored in MongoDB as a standalone document? If so, no problem, but if not, you may have to migrate another developer's DB.

Reply:
It was a document